tasmota/include/tasmota_types.h

```diff
@@ -180,7 +180,7 @@ typedef union {                            // Restricted by MISRA-C Rule 18.4 bu
   uint32_t data;                           // Allow bit manipulation using SetOption
   struct {                                 // SetOption146 .. SetOption177
     uint32_t use_esp32_temperature : 1;    // bit 0  (v12.1.1.1) - SetOption146 - (ESP32) Show ESP32 internal temperature sensor
-    uint32_t spare01 : 1;                  // bit 1
+    uint32_t mqtt_disable_sserialrec : 1;  // bit 1  (v12.x.x.x) - SetOption147 - (MQTT) Disable publish SSerialReceived MQTT messages, you must use event trigger rules instead.
```

Comment:
Can you please add the current version of Tasmota?

Reply:
Ok, I didn't know if I need to put the actual version or the next version. Thanks.